sidecar/sidecar.py

```diff
@@ -65,18 +83,22 @@ def watchForChanges(label, targetFolder, url, method, payload, current):
             for filename in dataMap.keys():
                 print("File in configmap %s %s" % (filename, eventType))
                 if (eventType == "ADDED") or (eventType == "MODIFIED"):
-                    writeTextToFile(targetFolder, filename, dataMap[filename])
+                    writeTextToFile(destFolder, filename, dataMap[filename])
                     if url is not None:
                         request(url, method, payload)
                 else:
-                    removeFile(targetFolder, filename)
+                    removeFile(destFolder, filename)
                     if url is not None:
                         request(url, method, payload)
 
 
 def main():
     print("Starting config map collector")
     label = os.getenv('LABEL')
+    # I want this to be configurable, but I also don't want to hack up
+    # the upstream grafana chart right now. Hard coding this for now...
+    # folderAnnotation = os.getenv('FOLDER_ANNOTATIONS')
+    folderAnnotation = "k8s-sidecar-target-directory"
```

Comment:
I am against hardcoding it like this here, why don't you use it as a default value in case the environment variable doesn't exist?

Reply:
good idea